fastlane/Fastfile

```diff
@@ -546,12 +570,12 @@ ENV["validate_translations"]="lintVanillaRelease"
     Dir.chdir("..") do
       sh("mkdir -p #{build_dir}")
       unless is_ci
-        # We pre-build the Gutenberg RN bundle on a different job on CI,  
-        # so cleaning here breaks the build 
+        # We pre-build the Gutenberg RN bundle on a different job on CI,
+        # so cleaning here breaks the build
         UI.message("Cleaning branch...")
         sh("echo \"Cleaning branch\" >> #{logfile_path}")
         sh("./gradlew clean >> #{logfile_path} 2>&1")
-      end 
```

Comment:
I was just skimming through this PR again and noticed this. I don't think we need this `unless is_ci` logic anymore since there is no CI job to bundle the JS files.

Reply:
Right, but also, we don't need it on CI anyway because we always start from a clean machine 🤷‍♂️. It just saves up a bit of time. But if you want to remove it, I'm fine with this.

Reply:
I don't mind keeping the code, but I think we should definitely change/remove the comment because it's referring to an action that no longer exists. Looking at the `fastlane/Fastfile`, there are other things that needs to be updated. For example, we can probably remove the `gutenberg_dep_check` lane. We could handle these in a separate PR.

Reply:
Yeah! Thinking more about it, it occurs to me that enabling clean-up on CI is not that bad. If we build more than one scheme in the same machine, it assures that we always start from a clean state. 
I'll open a new PR to address this.

Reply:
Done here: https://github.com/wordpress-mobile/WordPress-Android/pull/14337